jmclient/jmclient/wallet.py

```diff
@@ -635,7 +648,7 @@ def inputs_consumed_by_tx(self, tx):
         wallet, and returns [(index, CTxIn),..] for each.
         """
         retval = []
-        for i, txin in len(tx.vin):
+        for i, txin in enumerate(tx.vin):
```

Comment:
Thanks for the bugfix :) I can't remember where I thought this code was useful, but apparently it's not currently used.

Reply:
I must have encountered and fixed it when I was trying to use this method in a previous iteration.